server/pbenchinacan/load_keycloak.sh

```diff
@@ -53,7 +62,8 @@ echo
 echo "Keycloak connection successful on : ${KEYCLOAK_HOST_PORT}"
 echo
 
-status_code=$(curl -f -s -o /dev/null -w "%{http_code}" -X POST "${KEYCLOAK_HOST_PORT}/admin/realms" \
+status_code=$(curl -f -s -o /dev/null -w "%{http_code}" -X POST \
+  --cacert "${PB_DEPLOY_FILES}/pbench-server.crt" "${KEYCLOAK_HOST_PORT}/admin/realms" \
   -H "Authorization: Bearer ${ADMIN_TOKEN}" \
   -H "Content-Type: application/json" \
```

Comment:
Assuming that you don't remove the `--cacert` option here (which I think you should), please put the option on its own line, like the rest of the lines, here (and maybe put the URL at the end of the arguments list).